README.md

```diff
@@ -239,8 +239,8 @@ Listens to changes that are on the optional `range`.
 `watcher.destroy()`\
 Closes the watcher.
 
-`watcher.on('change', (newVersion, oldVersion) => {})`\
-Emitted after a feed change.
+`watcher.on('change', (current, previous) => {})`\
```

Comment:
Kind of makes more sense to have the old/previous first `(previous, current)`?

Reply:
current first is more natural to me

Reply:
Now it's an object so order is irrelevant, should we revert to yield an array?